script/prepare-release.js

```diff
@@ -95,19 +97,44 @@ async function getReleaseNotes (currentBranch) {
       `${currentBranch}, skipping release.`)
     process.exit(0)
   }
+
+  let prCount = 0
   commitComparison.data.commits.forEach(commitEntry => {
     let commitMessage = commitEntry.commit.message
-    if (commitMessage.toLowerCase().indexOf('merge') > -1) {
-      let mergeRE = /Merge pull request (#\d+) from .*\n/
-      let prMatch = commitMessage.match(mergeRE)
-      commitMessage = commitMessage.replace(mergeRE, '').replace('\n', '')
-      if (commitMessage.substr(commitMessage.length - 1, commitMessage.length) !== '.') {
-        commitMessage += '.'
+    try {
+      if (commitMessage.toLowerCase().indexOf('#') > -1) {
```

Comment:
Is `.toLowerCase()` needed in this line?

Reply:
@ckerr I'm looking for the lower case # 😉